eventsource/eventsource-onmessage-realm.htm

```diff
@@ -5,14 +5,14 @@
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
 
-<iframe src="eventsource-onmessage-realm-support.htm"></iframe>
+<iframe src="resources/eventsource-onmessage-realm.htm"></iframe>
 
 <script>
 "use strict";
 
 async_test(t => {
   window.onload = t.step_func(() => {
-    const source = new frames[0].EventSource("resources/message.py");
```

Comment:
Did you run this particular change through all browsers? It strikes me they might not use the correct global to parse URLs against.

Reply:
Nice eye! I verified this change in Firefox and Chromium; it does not effect the results reported by either browser for the affected tests. I'm not able to verify in Safari, though.